episodes/chapter4.md

```diff
@@ -0,0 +1,244 @@
+4. Click on the `Submit` button to create the new user.
+
+![Create a new user](fig/create_user.png)
+
+### Create a new collaboration
+A collaboration is a collection of one or more organizations. Let's create a new collaboration between two organizations. Here are the steps:
+
+1. Click on the `Collaborations` tab in the administration page.
+2. Click on the `Create collaboration` button.
+3. Fill in the details of the new collaboration.
+    * For `Encrypted` box, you can select whether the collaboration should be encrypted or not.
+    * You can select the organizations that will participate in the collaboration by selecting them from the `Organizations` dropdown.
+    * By default, we select the `Register nodes` box. This will ensure the nodes of the organizations are registered in the collaboration. If you don't select this box, you will have to register the nodes manually later.
+
+![Create a new collaboration](fig/create_collab_01.png)
+
+4. Click on the `Submit` button to create the new collaboration.
+    * After submitting the form, you may see a dialog box to ask you to download the API key (on MacOS) or a dialog points out that the API key has been downloaded (on Windows). The API key is used to authenticate the nodes in the collaboration. Later we will see how to reset the API key for a node.
+
+![Download API keys](fig/create_collab_02.png)
+
+You will see a message:
+
+> API key download
+>
+> The API keys have been downloaded.
+>
+> Please distribute each of these keys privately to each of the organizations. Note that they may reset their API key so that no-one but them knows it.
+
+5. After creating the collaboration, you can see the details of the collaboration by clicking on the specific collaboration listed in the `Collaborations` tab.
+    * You will see what organizations are participating in the collaboration.
+    * You will see the nodes of the organizations that are registered in the collaboration. If no nodes are registered, you can register them manually by clicking the `Register missing nodes` button.
+    * You can also see the algorithm store available for the collaboration. You can add a algorithm store for the collaboration by clicking the `Add algorithm store` button.
+    * You can also see the studies of the collaboration. You can add a study by clicking the `Add study` button.
+
+![Collaboration details](fig/create_collab_03.png)
+
+### Reset API key for a node
+If you want to reset the API key for a node, you can do so by following these steps:
+
+1. Click on the `Nodes` tab in the administration page.
+2. Click on the tab of the node you want to reset the API key for in the list of nodes.
+
+![Reset API key for a node](fig/reset_api_key_01.png)
+
+3. Click on the `Reset API key` button.
+    * You will see a dialog box to ask you to download the new API key.
+
+![Download new API key](fig/reset_api_key_02.png)
+
```

Comment:
Based on my previous comment, I think more context should be given on what the implications of resetting the key are, and why or under which circumstances you would do that.